sklearn/tests/test_naive_bayes.py

```diff
@@ -752,6 +752,17 @@ def test_alpha():
                          X, y, classes=[0, 1])
 
 
+def test_check_alpha():
```

Comment:
please also test that appropriate warnings are raised in each case (use pytest.warns)